MEME/MEME/Artist/ArtistHome/ArtistHomeView/ArtistHomeViewController.swift

```diff
@@ -213,7 +219,59 @@ class ArtistHomeViewController: UIViewController {
         }
         return result.replacingOccurrences(of: "_", with: ":")
     }
+    
+}
+// MARK: - UITableViewDataSource
+extension ArtistHomeViewController : UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if let reservationData = reservationData {
+            fromTomorrowCount = 0
+            showDataCount = 0
+            for i in 0..<reservationData.count {
+                self.showDataCount += 1
+                if reservationStatusData[i] == 1{
+                    fromTomorrowCount += 1
+                    if fromTomorrowCount == 2 {
+                        break
+                    }
+                }
+            }
+            return self.showDataCount
+        }else {
+            return 0
+        }
```

Comment:
fromTomorrowCount가 최대 2까지 가능할 것 같은데요, 홈 화면에서 예약 현황을 최대 2개까지 보여주기 위함일까요?
reservationData에서 오늘을 제외한 최근 예약 중 2개를 보여줘야 하는 것으로 알고 있습니다!
그렇다면 
`func fetchRecentReservation() -> [예약 정보 구조체] {}`
함수를 만들고, func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int 함수에서 fetchRecentReservation로 가져온 배열의 count 만큼 return 하는건 어떨까요?

```suggestion
extension ArtistHomeViewController : UITableViewDataSource {
    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
       guard let reservationData = reservationData else { return 0 }
       let recentReservations = fetchRecentReservation()
       return recentReservations.count
    }
}

//기타 VC 코드

    func fetchRecentReservation() -> [예약 정보 구조체] {
       //단, 배열은 날짜 순 정렬되어 있어야 함(최근 예약이 앞에 오도록)
        let recentReservationDatas = reservationData.filter { $0.date != 오늘 날짜 }
        return Array(recentReservationDatas.prefix(2))
    }
```